### ISSUE TITLE
background images data set size

### ISSUE THREAD

**Issue by shliang0603** [CDOTAD/AlphaGAN-Matting#3]
@CDOTAD Hi, I have applied for the Adobe data set, Is your data set processing combined with COCO data ?

**Comment by CDOTAD** [comment]
Yes, I used the COCO dataset as the background. I remembered that there is a list of the COCO images that the author used. But I didn't use that list. I just chose some images with on strategies.
